lib/yarp/version.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module YARP
+  VERSION = "0.8.0"
```

Comment:
Is it required for gemspec because of the `require_relative 'lib/yarp/version'` added in this PR? I wonder if we can avoid needing this file after `rm -rf lib ext` if we don't require it from the gemspec

Reply:
But how would the gemspec get the version then? Duplicate the version there? It sounds rather error-prone.

Reply:
Maybe I am misunderstanding. It looks like it is duplicated in the gemspec in this branch as well already.

Reply:
Ah that's a bug, I must have tried to hardcode it to let `bundle install` work locally (when the version was in template.rb), I'll fix.

Reply:
Done